modules/spacecraftinstruments/rendering/renderableplanetprojection.h

```diff
@@ -79,12 +79,12 @@ class RenderablePlanetProjection : public Renderable {
 
     ghoul::opengl::ProgramObject* _programObject = nullptr;
     ghoul::opengl::ProgramObject* _fboProgramObject = nullptr;
-    UniformCache(sunPos, modelTransform, modelViewProjectionTransform, hasBaseMap,
+    UniformCache(sun_pos, modelTransform, modelViewProjectionTransform, hasBaseMap,
         hasHeightMap, heightExaggeration, meridianShift, ambientBrightness,
-        projectionFading, baseTexture, projectionTexture, heightTexture)
-        _mainUniformCache;
+        projectionFading, baseTexture, projectionTexture,
+        heightTexture) _mainUniformCache;
 
-    UniformCache(projectionTexture, projectorMatrix, modelTransform, boresight,
+    UniformCache(projectionTexture, ProjectorMatrix, ModelTransform, boresight,
```

Comment:
Why are the first letters in caps?

Reply:
Same for the comment below as well.

I didn't want to introduce any additional bugs if necessary, so for those cases where the variable name in the UniformCache (in this case `projectorMatrix`) disagreed with the uniform name in the shader (in this case `ProjectorMatrix`), I chose to replace the variable name, is a mismatch would cause a compiler error, while changing the shader code to have a better uniform name was a more brittle change.

I'd see going through the individual shaders and cleaning them up as a bigger change that we'd want to more thoroughly test